src/transformers/quantizers/auto.py

```diff
@@ -47,6 +49,7 @@
     "quanto": QuantoHfQuantizer,
     "eetq": EetqHfQuantizer,
     "hqq": HqqHfQuantizer,
+    "compressed-tensors": CompressedTensorsHfQuantizer,
```

Comment:
@SunMarc made this change - but it uncovered that since `compressed-tensors` is already used elsewhere (ie vllm) with the name `compressed-tensors`, this would be a breaking change. Would it be possible to keep the `-`? Could leave an in-line comment if that helps

Reply:
Let's leave it like that then if this will be breaking !